### DISCUSSION TITLE
Are there benchmarks or usually processing time informations?

### DISCUSSION THREAD

**Discussion by AndreKelling** [halaxa/json-machine#103] [Q&A]
I am very glad founding this nice package of code!
I am the first time parsing a bigger JSON into PHP.

----

I am wondering what are usually parsing times and was looking for some benchmarks on the net.
Just did not found something useful: https://www.google.com/search?q=parse+json+php+benchmark

Maybe it's normal that a 25mb sized with 60k items JSON needs a minute to parse?! 
I guess it should just take several seconds, but i am not very sure... (it now takes just a few seconds on my site ;)

It all depends on the machines, of course.

Maybe there are some "insights" available?

**Comment by halaxa** [comment]
Hello. Better late then never 😉 JSON Machine has its own built in method for checking performance while developing. Just run `make performance-tests`. If you're looking for some solution to cross check different decoders - I don't know about general solution to this. One minute for 25mb json seems very slow - check for disabled xdebug, enabled opcache or jit.
